### ISSUE TITLE
Annotation toggling 

### ISSUE THREAD

**Issue by simonwibberley** [simonwibberley/old-bailey-map#8]
Why is it possible to toggle on/off - in the blue panel of annotations, displayed above the results - some annotations but not others (eg  crime type)?

**Comment by simonwibberley** [comment]
@alxbutterworth could you elaborate? Do you mean why do some toggles add a column, and others not?

**Comment by alxbutterworth** [comment]
Yes. Just so. On reflection, I imagine that it may be because the eg crime types are annotations on the trial rather than the individual sentences. Is that so and, if so, could they be applied to all sentences within trials?
